docs/user-guide/federation/index.md

```diff
@@ -46,3 +46,14 @@ The following guides explain some of the resources in detail:
 
 [API reference docs](/federation/docs/api-reference/readme/) lists all the
 resources supported by federation apiserver.
+
+## Cascading deletion
+
+Kubernetes v1.5 includes alpha support for cascading deletion of federated
+ingresses, namespaces, replica sets, secrets, deployments and daemon sets
+using `orphanDepedents` delete options.
```

Comment:
It is a run-on sentence. Better to restructure as follows:

"Kubernetes version 1.5 includes support for cascading deletion of federated resources. With cascading deletion, when you delete a resource from the federation control plane, the corresponding resources on all underlying clusters are also deleted.

To enable cascading deletion, set the option `DeleteOptions.orphanDependents=false` when you delete a resource from the federation control plane.

The following Federated resources are affected by cascading deletion:

* Ingress
* Namespace
* ReplicaSet
* Secret
* etc"